magefile.go

```diff
@@ -41,10 +41,14 @@ func Lint() error {
 	if err := sh.RunV("goimports", "-w", "-l", "."); err != nil {
 		return err
 	}
+	if err := sh.RunV("git", "diff", "--exit-code"); err != nil {
+		return err
+	}
+	// Run after git diff check because it produces random changes different from when run locally
```

Comment:
💯

Reply:
Had to change it again. go.mod changes also. Weird.